### ISSUE TITLE
Nodes in backlinks explorer expose the wrong accessible label for screen readers

### ISSUE THREAD

**Issue by Neurrone** [kortina/vscode-markdown-notes#67]
**Describe the bug**
File name nodes in the backlinks explorer are not reading correctly with the NVDA screen reader.

**To Reproduce**

1. On a note that has backlinks / imbound references, move focus to the backlinks explorer with the NVDA screen reader running.
2. Use the arrow keys to interact / navigate the tree view. NVDA announces the currently focused item as you move about with the keyboard.

**Expected behavior**

Notice that when focus is on file nodes, it reads "2 references" and the file name is not announced. I expected it to read something like "my-note.md: 3 references"

**Versions**
 - OS: Windows 10 2004
 - VS Code Version: 1.48.0-insider
 - Extension Version: 0.0.13

**Comment by kortina** [comment]
Does this need to use the same `item.label` attribute or is there a separate label for accessibility, so we can leave the visual label as is and just modify what the screen reader reads?

**Comment by Neurrone** [comment]
There should be a separate label for accessibility. 

I'm not familiar with the Vs Code API extensions API, which API is being used to construct this tree? There should be a way of providing an alternative label for this purpose. Otherwise, I can report it to the right people who can get it fixed in Vs Code.

**Comment by lukesmurray** [comment]
From the vscode 1.47 release notes.

> We have finalized the AccessibilityInformation in the TreeItem, StatusBarItem, and TimelineItem elements to support screen readers. AccessibilityInformation contains a label and a role. The label is read out by a screen reader once the item has focus. role of the item defines how a screen reader interacts with it. The role should be set in special cases when, for example, a tree-like element behaves like a checkbox. If role is not specified, VS Code will pick the appropriate role automatically.

I'm not completely sure how to fix this but the additional information here may be helpful

**Comment by Neurrone** [comment]
Could anyone point me to the place in the code that renders this? I can find out more if I know where to look.

**Comment by Neurrone** [comment]
Bump

**Comment by lukesmurray** [comment]
I would check out the [tree data provider documentation](https://code.visualstudio.com/api/extension-guides/tree-view#tree-data-provider) from the vscode docs and then the [BackLinksTreeDataProvider](https://github.com/kortina/vscode-markdown-notes/blob/master/src/BacklinksTreeDataProvider.ts) within the code base. Hopefully that is enough to get you started?

**Comment by Neurrone** [comment]
Filed [this issue in Vs Code](https://github.com/microsoft/vscode/issues/108480), suspecting this is a bug in Vs Code itself.

**Comment by JacksonKearl** [comment]
VS Code team here, the aria label of a tree item is set primarily by the `accessibilityInformation` property of the tree item, or if that does not exist then the `tooltip`, otherwise the `{label} {description}`. By explicitly setting the tooltip to be just the description here: https://github.com/kortina/vscode-markdown-notes/blob/d43a66095ef2d071c52ece7363fd0fd425db8043/src/BacklinksTreeDataProvider.ts#L155-L157
the aria label also becomes just the description. You could resolve this by either setting the tooltip to be both the label and description, setting `accessibilityInformation`, or not setting a `tooltip` at all.
